### PR TITLE
Docs: improve the accuracy of the sqlite3.connect() timeout param

### PR DESCRIPTION
**Pull request by erlend-aasland**

### CODE REVIEW

**Comment by erlend-aasland** [comment]
Relevant SQLite docs: https://www.sqlite.org/c3ref/busy_timeout.html

File: Doc/library/sqlite3.rst [hunk 1]
```diff
@@ -272,9 +272,9 @@ Module functions
 
    :param float timeout:
        How many seconds the connection should wait before raising
-       an exception, if the database is locked by another connection.
-       If another connection opens a transaction to modify the database,
-       it will be locked until that transaction is committed.
+       :exc:`OperationalError`, when a table is locked.
```
Comment (CAM-Gerlach) [inline Doc/library/sqlite3.rst]:
```suggestion
       an :exc:`OperationalError` when a table is locked.
```

Minor verbiage tweak

**Review by CAM-Gerlach — COMMENTED** [review]
Very minor phrasing tweak, otherwise LGTM @erlend-aasland !

**Comment by erlend-aasland** [comment]
Thanks for the review, CAM!

**Review by CAM-Gerlach — APPROVED** [review]
Thanks for the your work, @erlend-aasland ! LGTM from me.

**Pull request merged**